js/util/helpers.js

```diff
@@ -882,6 +882,55 @@ define(["dao", "globals", "lib/knockout", "util/eventLog"], function (dao, g, ko
         return (arg > 0 ? "+" : "") + round(arg, d);
     }
 
+    /**
+     * Constructor for a sorter object. Use the objects sortF member as the sort
+     * function for the object list.
+     *
+     * @param {Array of String fields} sortBy sorting fields in order of application
+     */
+    function MultiSort(sortBy) {
+
+        sortBy = sortBy;
+
+        this.sortF = function (a, b) {
+            var result, sortT, rev, i;
+
+            for (i = 0; i < sortBy.length; i++) {
+                if (sortBy[i].indexOf("-") === 0) {
+                    rev = true;
+                    sortT = sortBy[i].slice(1);
+                } else {
+                    rev = false;
+                    sortT = sortBy[i];
+                }
+
+                result = (rev) ? a[sortT] - b[sortT] : b[sortT] - a[sortT];
+
+                if (result || i === sortBy.length - 1) {
+                    return result;
+                }
+            }
+        };
+    }
+
+    /**
+     * Determine HCA advantage for this matchup. Team with better winp, cwinp,
+     * ocwinp and diff will get HCA regardless of seeding.
+     */
+    function seriesHomeAway(series, teamsConf, seed1, seed2, order, cid) {
```

Comment:
Would be good to have unit tests for this, to make sure it's always working 100% for all edge cases.